packages/test-utils/src/wrapper.js

```diff
@@ -15,6 +15,12 @@ import { matches } from './matches'
 import createDOMEvent from './create-dom-event'
 import { throwIfInstancesThrew } from './error'
 
+const wrapperInstances = []
```

Comment:
This needs to be an array because when passing a hook function it can happen that multiple wrapper instances were created.